src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -113,34 +113,40 @@ Filename: `migrations/1_initial_migration.js`
 ```javascript
 var Migrations = artifacts.require("Migrations");
 
-module.exports = function(deployer) {
-  // Deploy the Migrations contract as our only task
-  deployer.deploy(Migrations);
+module.exports = function(deployer, networks, accounts) {
+  deployer.then(async () => {
```

Comment:
This is incorrect, `deployer` is an object with a `deploy` function on it. However, as a side note you can make the entire function exported by each migration file `async` and use `await` here if you like. So like
```
module.exports = async function(deployer, network, accounts) {
  await deployer.deploy(Migrations);
  //... the rest of the file
```
Also note that the second argument is the network (singular - it is the name of the network) that is being used at the time.